lib/importer/views/plugin_config.html

```diff
@@ -93,7 +93,40 @@ <h2 class="govuk-heading-l">Column settings</h2>
       </div>
 
     </div>
-  </div>
+      <form action="config" method="post">
+
+    <div class="govuk-grid-column-full govuk-!-padding-top-6" >
+        <h2 class="govuk-heading-l"></h2>
+
+<div class="govuk-form-group">
+  <fieldset class="govuk-fieldset" aria-describedby="waste-hint">
```

Comment:
The aria attribute should be the id of an element which describes this one. If we don't have one then we can just remove the attribute?

Reply:
oops copy and paste fail on my part

Reply:
Added to 97ef5d9